pkgs/development/libraries/gobject-introspection/wrapper.nix

```diff
@@ -28,5 +28,22 @@ gobject-introspection-unwrapped.overrideAttrs (previousAttrs: {
       chmod +x "$dev/bin/g-ir-compiler"
       chmod +x "$dev/bin/g-ir-scanner"
     )
+  ''
+  # when cross-compiling and using the wrapper then when a package looks up the g_ir_X
+  # variable with pkg-config they'll get the host version which can't be run
+  # override the variable to use the absolute path to g_ir_X in PATH which can be run
+  + ''
+    cat >> $dev/nix-support/setup-hook <<-'EOF'
```

Comment:
changing `EOF` to `'EOF'` fixed the issue with librsvg, it was replacing the type -p g-ir-scanner during the cat